components/raftstore/src/store/peer.rs

```diff
@@ -1902,10 +1902,14 @@ where
                         &metrics.wf_commit_not_persist_log
                     };
                     for tracker in trackers {
-                        tracker.observe(now, hist, |t| {
-                            t.metrics.commit_not_persisted = !commit_persisted;
-                            &mut t.metrics.wf_commit_log_nanos
-                        });
+                        // Collect the metrics related to commit_log
+                        // durations.
+                        metrics
```

Comment:
It's more clear to separate into two lines as the track.observe is not just return a value, but it changes the variable wf_commit_log_nanos.

```suggestion
       let dur_nanos = tracker.observe(now, hist, |t| {	                        // Collect the metrics related to commit_log
                            t.metrics.commit_not_persisted = !commit_persisted;	                        // durations.
                            &mut t.metrics.wf_commit_log_nanos);
                            
           metrics.stat_commit_log.record(Duration::from_nanos(dur_nanos);
```